src/north/console/Console.class.js

```diff
@@ -47,6 +51,8 @@ class Console extends ApiHandler {
       fileSize,
     }]
     console.table(data)
+    this.statusData['Last uploaded file'] = filePath
```

Comment:
Can you add a 'Last upload at' to display the date ? Please do so everywhere you have the same logic